### PR TITLE
Playground: support fields tooltip

### PR DESCRIPTION
**Pull request by Mahmoud-Emad**
### Description

This pull request introduces a new feature to the playground that allows the addition of tooltips or hints on all fields used within the application. The tooltips provide helpful information and guidance to users, improving the overall user experience.

### Changes

- Implemented a tooltip/hint input component for all fields in the playground.
- Updated the field rendering logic to display the tooltip icon and handle tooltip visibility.

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/538
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/522

### Screenshots
- image
- image
- image

### Checklist

- [ ] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by xmonader — APPROVED** [review]

**Comment by xmonader** [comment]
please resolve the conflicts

File: packages/playground/src/components/k8s_worker.vue [hunk 1]
```diff
@@ -50,12 +69,42 @@
...
+      />
+    </input-tooltip>
   </input-validator>
 
-  <v-switch label="Public IPv4" inset color="primary" v-model="$props.modelValue.ipv4" />
-  <v-switch label="Public IPv6" inset color="primary" v-model="$props.modelValue.ipv6" />
-  <v-switch label="Planetary Network" inset color="primary" v-model="$props.modelValue.planetary" />
+  <v-tooltip
+    location="top"
+    text="Public IPv4 refers to an Internet Protocol version 4 address that is globally unique and accessible over the internet."
```
Comment (AhmedHanafy725) [inline packages/playground/src/components/k8s_worker.vue:84]:
```suggestion
    text="An Internet Protocol version 4 address that is globally unique and accessible over the internet."
```

File: packages/playground/src/components/root_fs_size.vue [hunk 2]
```diff
@@ -17,7 +17,9 @@
       </input-validator>
     </div>
 
-    <v-tooltip text="Toggle editing root filesystem size">
+    <v-tooltip
+      text="The 'Toggle Editing Root Filesystem Size' feature allows users to adjust the size of the root filesystem for their system, The root filesystem is the primary storage location for the operating system and its associated files."
```
Comment (AhmedHanafy725) [inline packages/playground/src/components/root_fs_size.vue:21]:
```suggestion
      text="Allows adjusting the size of the root filesystem. The root filesystem is the primary storage location for the operating system and its associated files."
```

File: packages/playground/src/weblets/full_vm.vue [hunk 3]
```diff
@@ -77,13 +96,55 @@
...
+          text="The Planetary Network is a distributed network infrastructure that spans across multiple regions and countries, providing global connectivity."
+        >
+          <template v-slot:activator="{ props }">
+            <v-switch color="primary" inset label="Planetary Network" v-model="planetary" v-bind="props" />
+          </template>
+        </v-tooltip>
+
+        <v-tooltip
+          location="top"
+          text="Enabling WireGuard Access allows you to establish secure and encrypted connections to your network resources."
```
Comment (AhmedHanafy725) [inline packages/playground/src/weblets/full_vm.vue:141]:
```suggestion
          text="Enabling WireGuard Access allows you to establish private,  secure, and encrypted connections to your instance."
```

File: packages/playground/src/components/algorand_capacity.vue [hunk 4]
```diff
@@ -15,7 +19,14 @@
         #="{ props }"
         ref="cpuInput"
       >
-        <v-text-field label="CPU (vCores)" type="number" v-model.number="cpu" v-bind="props" />
+        <input-tooltip #="{ tooltipProps }" tooltip="the number of virtual cores allocated to your solution.">
```
[the remark this answers is not in the document]
Comment (Mahmoud-Emad) [reply packages/playground/src/components/algorand_capacity.vue:22]: applied and used

File: packages/playground/src/weblets/full_vm.vue (hunk 3 above)
Comment (Mahmoud-Emad) [reply packages/playground/src/weblets/full_vm.vue]: should be updated

File: packages/playground/src/components/root_fs_size.vue (hunk 2 above)
Comment (Mahmoud-Emad) [reply packages/playground/src/components/root_fs_size.vue]: should be updated

File: packages/playground/src/components/k8s_worker.vue (hunk 1 above)
Comment (Mahmoud-Emad) [reply packages/playground/src/components/k8s_worker.vue]: should be updated

File: packages/playground/src/components/algorand_capacity.vue [hunk 5]
```diff
@@ -41,7 +62,17 @@
         #="{ props }"
         ref="storageInput"
       >
-        <v-text-field label="Storage Size (GB)" type="number" v-model.number="storage" v-bind="props" />
+        <input-tooltip
+          #="{ tooltipProps }"
+          tooltip="Disk Size (GB) refers to the storage capacity allocated to your solution, indicating the amount of space available to store files, data, and applications."
```
[the remark this answers is not in the document]
Comment (Mahmoud-Emad) [reply packages/playground/src/components/algorand_capacity.vue]: should be updated

File: packages/playground/src/components/algorand_capacity.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (Mahmoud-Emad) [reply packages/playground/src/components/algorand_capacity.vue]: should be updated

File: packages/playground/src/components/algorand_capacity.vue [hunk 6]
```diff
@@ -28,7 +39,17 @@
         #="{ props }"
         ref="memoryInput"
       >
-        <v-text-field label="Memory (MB)" type="number" v-model.number="memory" v-bind="props" />
+        <input-tooltip
+          #="{ tooltipProps }"
+          tooltip="Memory (MB) refers to the amount of RAM (Random Access Memory) allocated to your solution."
```
[the remark this answers is not in the document]
Comment (Mahmoud-Emad) [reply packages/playground/src/components/algorand_capacity.vue]: should be updated

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**